# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/reftests/css-display/reftest.list
# Commit: 867abe6c80a3
# Full Hash: 867abe6c80a36d66fec92f276c3a7d7c315637bf
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-03-01 09:36:12
# Description:
#   Bug 1694459 - Deal with 'display:contents' style changes on <fieldset> descendants better where it affects which <legend> is the rendered legend.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106566
# ==============================================================================

diff -r 77247863a2ea -r 867abe6c80a3 layout/reftests/css-display/reftest.list
--- a/layout/reftests/css-display/reftest.list	Mon Mar 01 01:16:36 2021 +0000
+++ b/layout/reftests/css-display/reftest.list	Mon Mar 01 03:42:41 2021 +0000
@@ -16,7 +16,6 @@
 == display-contents-visibility-hidden.html display-contents-visibility-hidden-ref.html
 == display-contents-visibility-hidden-2.html display-contents-visibility-hidden-ref.html
 == display-contents-495385-2d.html display-contents-495385-2d-ref.html
-asserts(0-1) fuzzy-if(Android,0-8,0-3216) == display-contents-fieldset.html display-contents-fieldset-ref.html # bug 1089223
 == display-contents-list-item-child.html display-contents-list-item-child-ref.html
 == display-contents-dyn-insert-text.html display-contents-dyn-insert-text-ref.html
 == display-contents-writing-mode-1.html display-contents-writing-mode-1-ref.html